core/lib/libtomcrypt/sub.mk

```diff
@@ -116,7 +116,10 @@ srcs-$(_CFG_CORE_LTC_HMAC) += hmac.c
 srcs-$(_CFG_CORE_LTC_CMAC) += cmac.c
 srcs-$(_CFG_CORE_LTC_ECB) += ecb.c
 srcs-$(_CFG_CORE_LTC_CBC) += cbc.c
+ifeq ($(CFG_CORE_SE05X),n)
 srcs-$(_CFG_CORE_LTC_CTR) += ctr.c
+endif
+srcs-$(_CFG_CORE_LTC_AES) += aes.c
```

Comment:
remove this line.

Reply:
sorry what line? 122?

Reply:
122 yes. This change duplicates line 130 (line 127 as for tag 3.10.0).

Reply:
ok done.